rosdep/python.yaml

```
@@ -6515,6 +6515,22 @@ python3-flask-cors:
   ubuntu:
     '*': [python3-flask-cors]
     bionic: null
+python3-flask-jwt-extended:
+  debian:
+    bullseye: [python3-python-flask-jwt-extended]
+    pip:
+      packages: [flask-jwt-extended]
+    sid: [python3-python-flask-jwt-extended]
+  freebsd: [py-flask-jwt-extended]
+  nixos: [python39Packages.flask-jwt-extended]
+  opensuse: [python-flask-jwt-extended]
+  ubuntu:
+    focal: [python3-python-flask-jwt-extended]
+    hirsuite: [python3-python-flask-jwt-extended]
+    impish: [python3-python-flask-jwt-extended]
+    jammy: [python3-python-flask-jwt-extended]
```

```suggestion
    '*': [python3-python-flask-jwt-extended]
```

Please change the current key for focal an newer to a wildcard, removing the other identical ones.